pyproject.toml

```diff
@@ -10,23 +10,23 @@ authors = [
     { name = "Marius Müller", email = "marius.mueller@physik.uni-giessen.de" },
     { name = "Florian Dobener", email = "pyelli@schroedingerscat.org" },
 ]
-requires-python = ">=3.8"
+requires-python = ">=3.9"
 license = { file = "LICENSE.txt" }
 readme = "README.md"
 classifiers = [
     "Development Status :: 4 - Beta",
-    "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
     "Programming Language :: Python :: 3.11",
     "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
 ]
 dependencies = [
     "scipy",
     "numpy>=1.20",
     "numpy<2.0.0;python_version<'3.10'",
     "pandas>=1.0.0",
-    "h5py",
```

Comment:
We request this because only h5py>=3.12.1 supports python 3.13, right? Generally, I think it would be fine if lower python versions would install a lower h5py version and it should be constrained by the h5py python version constraints. So I think we could even lift this constraint. But maybe there are other reasons I'm not aware of. Anyways, I'm also ok with constraining the version (+ it's good to generally have a lower boundary).

Reply:
h5py seems to be unspecific with their supported versions and i tried to make both 3.8 and 3.13 work, but it should not be needed anymore.